catroid/src/main/res/values/strings.xml

```diff
@@ -1270,6 +1270,11 @@
     <string name="brick_web_request">Send web request to</string>
     <string name="brick_web_request_result">and store answer in</string>
     <string name="brick_web_request_default_value">https://official-joke-api.appspot.com/random_joke</string>
+    <string name="brick_store_csv_into_userlist_store">Store column #</string>
+    <string name="brick_store_csv_into_userlist_csv">of CSV</string>
```

Comment:
```suggestion
    <string name="brick_store_csv_into_userlist_csv">of the\ncomma separated values</string>
```

Let's make it clear. Please somehow add a line break in the layout as indicated, to make the lines more balanced. May need some additional tuning ;)